code/00_setup_environment.R

```diff
@@ -38,12 +38,57 @@ library(openxlsx)     # For reading and writing Excel files
 library(purrr)        # For iterative functional programming with vectors
 
 
-### 2 - Derive date variables
+### 2 - Derive date variables ----
 
 previous_qtr_end <- as_date(cut(ymd(current_qtr_end), "quarter")) - 1
 
 
-### 3 - Read all functions from the /functions directory ----
+### 3 - Set file paths ----
+
+# Path for copying in 14 quarterly submissions for most recent quarter
+path_submissions = "//PHI_conf/WaitingTimes/Chronic-Pain/Data/Database/Submissions & emails/Current Quarter/"
+
+# Path for copying in ALL DATA file from previous quarter
+path_alldata = "//PHI_conf/WaitingTimes/Chronic-Pain/Data/Database/"
+
+# Path for copying in lookup file for rate of referral calculation
+path_lookup = "//PHI_conf/WaitingTimes/Chronic-Pain/R development work/Lookups/Populationestimatemid2020_18+.xlsx"
+
+# Path for writing out the Board Narrative column for the workbook
+path_narrative =  here("data", "output")
+
+# Path for writing out current quarter dataframe
+path_currentquarter = here("data", "input")
+
+# Path for writing out updated ALL DATA file
+path_discovery =  here("data", "output")
+
+# Path for writing out file for Discovery
+path_newalldata = here("data", "output")
```

Comment:
You have 3 paths here that are all the same, abstract these to a single `path_output` variable.